MetaMask/metamask-extension
# Onboarding v2: Implement requested metrics

## Explanation

Implements the requested metrics events for the new onboarding flow.

## Manual Testing Steps

- Go through various onboarding flows, setting a breakpoint on `trackEvent` usage to ensure the values are correct.
- Look at https://www.notion.so/New-Extension-onboarding-v2-9e017a2c83e24cc3a2ecf2c561b1868a#ed9b49ad8e904e67a97482fdb5e9d34f to ensure all events are present
